docs/deprecations.md

```diff
@@ -6,6 +6,12 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+## Http Api provider tags
+
+Deprecation code: `AWS_HTTP_API_USE_PROVIDER_TAGS`
+
+Starting with v3.0.0, provider tags will be applied to Http Api Gateway by default
```

Comment:
Let's include information that it's possible to adapt to new behavior by setting `provider.httpApi.useProviderTags` to `true`

Reply:
updated the doc